curiosrc/web/static/chain-connectivity.mjs

```diff
@@ -60,8 +49,8 @@ window.customElements.define('chain-connectivity', class MyElement extends LitEl
         ${this.data.map(item => html`
         <tr>
             <td>${item.Address}</td>
-            <td>${item.Reachable ? html`<span class="success">ok</span>` : html`<span class="error">FAIL</span>`}</td>
-            <td>${item.SyncState === "ok" ? html`<span class="success">ok</span>` : html`<span class="warning">${item.SyncState}</span>`}</td>
+            <td>${item.Reachable ? html`<span class="alert alert-success">ok</span>` : html`<span class="alert altert-danger">FAIL</span>`}</td>
+            <td>${item.SyncState === "ok" ? html`<span class="alert alert-success">ok</span>` : html`<span class="alert alert-warning">${item.SyncState}</span>`}</td>
```

Comment:
Not sure about pulling in the whole boostrap just for those alert classes; They don't seem to fit too well to the rest of the styling

Reply:
Bootstrap is painting the top menu and soon will paint some other widgets. It's also doing most of the styling for the tables, so I'd rather not remove it until we have fixes for those.